test/integration/puppetlabs/services/certificate_authority/certificate_authority_int_test.clj

```diff
@@ -203,9 +203,7 @@
           subjectDN (ssl-utils/cn "test_cert_ca_true")
           serial 1
           public-key (ssl-utils/get-public-key key-pair)
-          ca-ext (ca/create-ca-extensions subjectDN
-                                          serial
-                                          public-key)
+          ca-ext [(ssl-utils/basic-constraints-for-ca)]
```

Comment:
Because `create-ca-extensions` now has a SAN extension attached, and that check with `allow-subject-alt-names` get's triggered before the check for the oid—2.5.29.19—for certificate signing.

Reply:
so you're saying that without this change, this test would fail because we specifically _don't_ allow subject alt names here? Still trying to make sense of what exactly this is testing.. 😅 

also thanks for letting me use your PRs to learn more about the various things we're responsible for 🙃

Reply:
yeah, remember how I made a mention in our team chat about how
> TIL the dev/puppetserver.conf affects `lein test`

There is a CA setting that is false for `allow-subject-alt-names` in that test conf file. I had turned it one to allow for additional testing in my local env, so I didn't see the errors that CI caught.

Because the test was using the `create-ca-extensions`, the way the list is formed is modified because we now ensure the SAN extension is there; that's what caused it fail with the same 409 status response code but a differnt body. The test always should have been more targeted to use a single extension, to avoid this exact situation.

Reply:
gotcha, thanks for explaining! Is there a reason we don't just set `allow-subject-alt-names: true` in the test conf? I don't have a sense of when someone would or would not want to allow SANs, but if we've added this functionality, it seems like most people must have `allow-subject-alt-names: true`?